ped_costmap:
    map_type: costmap
    global_frame: /map
    robot_base_frame: /base_link
    #map_topic: map_cost
    map_topic: /map
    static_map: true
    publish_frequency: 5
    #robot_radius: 1.2
    robot_radius: 0.9
    #inflation_radius: 7.0
    inflation_radius: 2.0
    #cost_scaling_factor: 4.0
    cost_scaling_factor: 4.0
    update_frequency: 2
    observation_sources: peds
    peds:
        topic: ped_prediction
        #topic: nothing
        clearing: true
        obstacle_range: 5000
        raytrace_range: 5000
    ped_cost_ratio: 0.9

planner:
    # steering_limit_deg: 18
    # yaw_res_deg: 1.0
    #steering_limit_deg: 48
    #yaw_res_deg: 2.0
    steering_limit_deg: 15
    yaw_res_deg: 2.5
    #cost_steering_deg: 30
    #cost_steering_deg: 8
    cost_steering_deg: 20
    step: 6
    planning_step_len: 2.0
    planning_step_len_res: 1.5
    num_search: 800000
    #discretize_ratio: 0.3
    discretize_ratio: 0.5
    discount: 0.99
#    car_len: 1.0
    car_len: 1.0

